src/Utils/csvUtils.ts

```diff
@@ -1,10 +1,10 @@
 export const csvGroupByColumn = (
   data: string,
-  groupByColumn: string,
+  groupByColumn: string = "Patient ID",
```

Comment:
why does "Patient ID" need to be a default for a generic shared utility function?
```suggestion
  groupByColumn: string,
```

Reply:
I thought we group them based in primary key (patient ID) by deafult when only data is passed as arg .

Reply:
but what if this util is used for assset exports or any other things?

Reply:
yeah thats true.